lib/models/locations_movies.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
Like @kienan mentioned earlier, you don't need this `LocationMovies` model since the `Location` and `Movies` Bookshelf models should use this under the hood. Do you mind removing this file and refactoring as necessary?